### ISSUE TITLE
Json to dbml, Note filed was losed .

### ISSUE THREAD

**Issue by Mrxyy** [holistics/dbml#485]
console.log(combined);
 const database = new Parser(undefined).parse(combined, 'json');
 console.log(database.normalize());
![image](https://github.com/holistics/dbml/assets/49010414/bce52709-239b-4f69-86cd-4a1b300944cd)
![image](https://github.com/holistics/dbml/assets/49010414/d9b04d35-26d0-45da-91b2-56d9a982b269)

**Comment by nguyenalter** [comment]
Hi @Mrxyy,

This is not an issue because the `field.note` is an object and not a string (changed in https://github.com/holistics/dbml/pull/300). 
You could change it to `note: { value: '...' }` to parse the note correctly.

By the way, did you get the `combined` value from the DBML exporter? If so, this is an exporter bug.

**Comment by Mrxyy** [comment]
Yes,I found some reasons. pegjs parsing results to JSON is no problem from DBML.
There is I recorded video below. 

https://github.com/holistics/dbml/assets/49010414/23638976-aaa0-4922-92cd-c995dff7828c

I am going fix it today.

**Comment by nguyenalter** [comment]
For more context, when exporting from the model structure to JSON, there are 2 note related data:
- `note`: string
- `noteToken`: pegjsToken, will look like this:
  ``` 
  {
    start: { offset: 23, line: 5, column: 6 },
    end:   { offset: 23, line: 5, column: 6 }
  }
  ```
However, when parsing (importing) the JSON to create the model structure, we only recognize the note in type of `{ value: string; token: pegjsToken }`.

It's best if you could improve the model structure to recognize these 2 note types:
1. The note is in type of ` { value, token }` (current behaviour)
2. The note is a `string` (exported result). For this case, you should set the correct noteToken value too.

**Comment by Mrxyy** [comment]
+ The difference between the "note" field in rawDatabase and the "note" field in database is that they have different data types.
     - In rawDatabase: {note: {value, token}}
     - In database instance: {noteToken,note}
   + All entities only accept note type is {note: {value, token}} when they were created. but note field type is {noteToken,note} in Json or database.These note field of entities will would as string type were used when ModelExporter.export api was executed.
   + So to need to promise type of note field is {noteToken, note} can be accepted in new entity. I have done these in the pr.
   + #487

**Comment by nguyenalter** [comment]
Thank you for submitting the issue and the pull request to address it.

This issue has been fixed in version 3.1.1.
